test/IntegrationTests/HttpTests.cs

```diff
@@ -82,8 +82,14 @@ public void SubmitMetrics()
     {
         using var collector = new MockMetricsCollector(Output);
         SetExporter(collector);
-        collector.Expect("OpenTelemetry.Instrumentation.Http");
         collector.Expect("OpenTelemetry.Instrumentation.AspNetCore");
+#if NET8_0_OR_GREATER
+        collector.Expect("System.Net.Http");
+        collector.Expect("System.Net.NameResolution");
+        collector.ExpectAdditionalEntries(x => x.All(m => m.InstrumentationScopeName != "OpenTelemetry.Instrumentation.Http"));
```

Comment:
Ensure that we do not fetched `OpenTelemetry.Instrumentation.Http`. It should be not enabled for .NET8 as in Instrumentation.Http package.